src/html.tsx

```diff
@@ -38,11 +43,14 @@ export const Report = ({ data }: ReportProps) => {
             sourceMap={sourceMap}
           />
         ))}
+        {pieChart && <script>{pieChartScript(pieChart)}</script>}
       </body>
     </html>
   );
 };
 
+const SHOW_STATS = false; // TODO: provide this as an option
```

Comment:
can I suggest we use google-style TODOs with usernames to leave a record of who has context for each one?

for this line that would look like
... // TODO(Onurbon): provide this as an option